pkg/referrerstore/oras/oras.go

```diff
@@ -231,11 +231,15 @@ func (store *orasStore) ListReferrers(ctx context.Context, subjectReference comm
 	}
 
 	if store.config.CosignEnabled {
-		cosignReferences, err := getCosignReferences(subjectReference, store.config)
+		// add cosign descriptor if exists
+		cosignReferences, err := getCosignReferences(ctx, subjectReference, store, repository)
```

Comment:
wonder if we need to run this block in parallel as `CosignEnabled` is always true now.

Reply:
This is left for certain use specific scenarios where the user only uses a non cosign verifier and for perf reasons want to manually disable the existence check